extensions/cornerstone/src/getCustomizationModule.ts

```diff
@@ -42,6 +43,14 @@ function getCustomizationModule() {
           id: 'cornerstone.windowLevelPresets',
           presets: defaultWindowLevelPresets,
         },
+        {
+          id: 'cornerstone.colorbar',
+          width: '2.5%',
+          colorbarTickPosition: 'left',
+          colormaps,
+          colorbarContainerPosition: 'right',
+          colorbarInitialColormap: 'Grayscale',
```

Comment:
DefaultColormap? or is it a name in cs3d?

Reply:
Done